src/server/server.js

```diff
@@ -22,10 +23,21 @@ class DEQMServer extends Server {
     this.app.get('/:base_version/file/:clientId/:fileName', configClientFile.clientFile);
     return this;
   }
+  async enableValidationMiddleWare() {
+    await axios.put('http://localhost:4567/igs/hl7.fhir.us.qicore');
+    this.app.put('/:base_version*', validateFhir);
+    this.app.post('/:base_version*', validateFhir);
```

Comment:
Can we be sure the async logic will work correctly here? I don't totally understand how the workflow of async middleware works

Reply:
This `PUT` is just loading QI core into the validator, right? Do we _need_ to do it in the declaration of this middleware? Can we not actually just do it in `index.js` when the server spins up? We do `await mongoClient.connect()` as an async thing after the server starts, can we just do it after that? 

Also, should use `process.env.VALIDATOR_HOST/PORT` heheh. I guess I'll just review this PR